wallet/wallet_test.go

```diff
@@ -205,3 +206,43 @@ func TestLabelTransaction(t *testing.T) {
 		})
 	}
 }
+
+// TestGetTransaction writes a tx to disk and then compares
+// the hash to the response from the GetTransaction function.
+func TestGetTransaction(t *testing.T) {
+	t.Parallel()
+
+	w, cleanup := testWallet(t)
+	defer cleanup()
+
+	// Create a new tx record.
+	rec, err := wtxmgr.NewTxRecord(
+		TstSerializedTx, time.Now(),
+	)
+	if err != nil {
```

Comment:
We are pushing the usage of `require` atm, so we wanna do `require.NoError` here and below.